docs/quickstart.ipynb

```diff
@@ -1555,35 +2072,36 @@
        "4               -6 -0.017603   0         0.0  6.918492  "
       ]
      },
```

Comment:
Should we maybe take out the DiD section from quickstart and simply link to the dedicated vignette?

Reply:
Yup, sounds good to me!

Reply:
Added in https://github.com/s3alfisc/pyfixest/pull/436/commits/55923fa4db0eb075e4be3395dc6e328b368a4da5